docs/developer-center/_internal/reference/07-CartoContext.md

```diff
@@ -20,7 +20,7 @@ There are two ways of authenticating against a CARTO account:
 
     ```python
     from cartoframes import Credentials
-    creds = Credentials(user='eschbacher', key='abcdefg')
```

Comment:
This has been replaced (for now) by the readthedocs version which has the correct `username` kwarg:
https://cartoframes.readthedocs.io/en/stable/cartocontext.html#cartocontext